expan/core/experiment.py

```diff
@@ -384,38 +380,33 @@ def bayes_precision_delta(self,
         Returns:
             a Results object
         """
-        try:
-            for mname in kpis_to_analyse:
-                metric_df = self._get_metric_df(mname, reference_kpis, weighted_kpis)
-
-                def do_delta(x, y, x_weights, y_weights):
-                    weighted_x = np.array(x, dtype=float) * x_weights
-                    weighted_y = np.array(y, dtype=float) * y_weights
-                    return early_stopping_to_dataframe(metric_df.columns[2],
-                                                       *es.bayes_precision(
-                                                           x=weighted_x,
-                                                           y=weighted_y,
-                                                           distribution=distribution,
-                                                           posterior_width=posterior_width,
-                                                           num_iters=num_iters))
-
-                df = metric_df.groupby('variant').apply(self._apply_reweighting_and_all_variants,
-                                                        metric_df=metric_df,
-                                                        weighted_kpis=weighted_kpis,
-                                                        reference_kpis=reference_kpis,
-                                                        mname=mname,
-                                                        func_apply_variants=do_delta).unstack(0)
-                # force the stop label of the baseline variant to 0
-                df.loc[(mname, '-', slice(None), 'stop'), ('value', self.baseline_variant)] = 0
-
-                if result.df is None:
-                    result.df = df
-                else:
-                    result.df = result.df.append(df)
-        finally:
-            # Remove .pkl compiled stan model files in the finally no matter script in try was successful or not
-            if remove_pkls:
-                remove_model_pkls()
```

Comment:
I just investigated this question, and now I think it should be safe to delete files from /var/folders/ because you're aware of files you created. Whether to delete compiled models or not depends on how often they will/should be changed or will they be changed at all in expan.